site/src/components/message.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Message
+description: The message component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - message
+---
+
+# Message
+
+Use message component for chat or messaging communications.
+
+## Usage
+
+Here is an example of a standard usage of message component.
+
+```html
+<article class="message">
+	<p class="message-time">May 29, 2023, 4:50PM</p>
+	<div class="persona message-persona">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="Avatar for Jane Doe" />
+		</figure>
+	</div>
+	<div class="message-content">
+		<div class="margin-bottom-xxs">
+			<div class="message-content-header">
+				<div class="display-grid align-items-center">
+					<p class="message-content-display-name">Q&A Moderator</p>
+					<p class="font-size-xs">Moderator</p>
+				</div>
+				<div class="message-content-options">
+					<span>⋮</span>
```

Comment:
Popover/button?

Reply:
Will said in the component make the markdown as simple as possible and in the pattern can make it as it should be for copy and paste.